### PR TITLE
HBW-88 Fix lookup and change multi select to single

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: hbw/app/javascript/packs/hbw/components/form/select.js.jsx [hunk 1]
```diff
@@ -166,11 +164,11 @@ modulejs.define('HBWFormSelect',
           });
       },
 
-      getDefaultValue (isMulti) {
+      getDefaultValue () {
         const variants = this.buildOptions();
 
         if (this.state.value) {
-          const value = (isMulti ? this.state.value : [this.state.value]).map(el => `${el}`);
+          const value = [this.state.value].map(el => `${el}`);
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/form/select.js.jsx]: You don't need to do this anymore I think.

**Comment by mxsmirnoff** [comment]
Updated

File: hbw/app/javascript/packs/hbw/components/form/select.js.jsx [hunk 2]
```diff
@@ -166,11 +164,11 @@ modulejs.define('HBWFormSelect',
...
 
           return variants.filter(opt => value.includes(`${opt.value}`));
```
[the remark this answers is not in the document]
Comment (TimAle) [reply hbw/app/javascript/packs/hbw/components/form/select.js.jsx]: Also check `setValue`, it tries to handle array too.

File: hbw/app/javascript/packs/hbw/components/form/select.js.jsx (hunk 1 above)
Comment (mxsmirnoff) [reply hbw/app/javascript/packs/hbw/components/form/select.js.jsx]: I thought `value` should contain only String

**Comment by mxsmirnoff** [comment]
Updated

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**